_includes/manuals/1.19/3.5.6_cli.md

```diff
@@ -0,0 +1,52 @@
+The Command Line Interface is based on the [hammer](https://github.com/theforeman/hammer-cli) framework. The foreman-related commands are defined in plugin ```hammer_cli_foreman```
+
+#### Format and locations
+
+Configuration is loaded from a set of directories in this order:
+
+ - ```./config/hammer/``` (config dir in CWD)
+ - ```/etc/hammer/```.
+ - ```~/.hammer/```
```

Comment:
The directory order is different now, see: https://github.com/theforeman/hammer-cli/blob/master/doc/installation.md#locations

Reply:
@tstrachota -thanks, can you please open a PR fixing this in the relevant manual versions?